// @vertx/core/options.js
/// <reference types="@vertx/core/options" />
module.exports = {
  AddressResolverOptions: Java.type('io.vertx.core.dns.AddressResolverOptions'),
  Argument: Java.type('io.vertx.core.cli.Argument'),
  CopyOptions: Java.type('io.vertx.core.file.CopyOptions'),
  DatagramSocketOptions: Java.type('io.vertx.core.datagram.DatagramSocketOptions'),
  DeliveryOptions: Java.type('io.vertx.core.eventbus.DeliveryOptions'),
  DeploymentOptions: Java.type('io.vertx.core.DeploymentOptions'),
  DnsClientOptions: Java.type('io.vertx.core.dns.DnsClientOptions'),
  EventBusOptions: Java.type('io.vertx.core.eventbus.EventBusOptions'),
  GoAway: Java.type('io.vertx.core.http.GoAway'),
  Http2Settings: Java.type('io.vertx.core.http.Http2Settings'),
  HttpClientOptions: Java.type('io.vertx.core.http.HttpClientOptions'),
  HttpServerOptions: Java.type('io.vertx.core.http.HttpServerOptions'),
  JdkSSLEngineOptions: Java.type('io.vertx.core.net.JdkSSLEngineOptions'),
  JksOptions: Java.type('io.vertx.core.net.JksOptions'),
  MetricsOptions: Java.type('io.vertx.core.metrics.MetricsOptions'),
  NetClientOptions: Java.type('io.vertx.core.net.NetClientOptions'),
  NetServerOptions: Java.type('io.vertx.core.net.NetServerOptions'),
  OpenOptions: Java.type('io.vertx.core.file.OpenOptions'),
  OpenSSLEngineOptions: Java.type('io.vertx.core.net.OpenSSLEngineOptions'),
  Option: Java.type('io.vertx.core.cli.Option'),
  PemKeyCertOptions: Java.type('io.vertx.core.net.PemKeyCertOptions'),
  PemTrustOptions: Java.type('io.vertx.core.net.PemTrustOptions'),
  PfxOptions: Java.type('io.vertx.core.net.PfxOptions'),
  ProxyOptions: Java.type('io.vertx.core.net.ProxyOptions'),
  RequestOptions: Java.type('io.vertx.core.http.RequestOptions'),
  VertxOptions: Java.type('io.vertx.core.VertxOptions')
};
